modules/nrfxlib/nrf_802154/sl/platform/nrf_802154_platform_sl_lptimer_grtc_pm_utils.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "nrf_802154_platform_sl_lptimer_grtc_pm_utils.h"
+
+#include <zephyr/drivers/timer/nrf_grtc_timer.h>
+#include <zephyr/kernel.h>
+#include <zephyr/pm/policy.h>
+
+#define LPTIMER_PM_UTILS_STACK_SIZE 256
+
+static K_SEM_DEFINE(m_lptimer_pm_utils_sem, 0, 1);
+
+static struct pm_policy_event m_pm_event;
+static int64_t                m_trigger_time;
+static bool		      m_event_registered;
+
+#define SYS_TICK_PER_GRTC_CYC \
+	((uint64_t)CONFIG_SYS_CLOCK_TICKS_PER_SEC / (uint64_t)sys_clock_hw_cycles_per_sec())
```

Comment:
I think this will be 0 in all cases
`CONFIG_SYS_CLOCK_TICKS_PER_SEC` is 10000
`sys_clock_hw_cycles_per_sec` returns 1000000

Reply:
Thanks, missed that

Reply:
Instead of using:
```c++
result * SYS_TICK_PER_GRTC_CYC
```
I will now use:
```c++
result \ GRTC_CYC_PER_SYS_TICK
```
while converting